### DISCUSSION TITLE
blog/the-legacy-of-zong-qinghou

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#39] [Announcements]
# blog/the-legacy-of-zong-qinghou

哲学片段与学术之路

https://pathos.page/blog/the-legacy-of-zong-qinghou

**Comment by jkjoy** [comment]
很多人会讲 "勿谈国事"
可是政治与民生息息相关.
我认为中国的资本与政治就是相互勾连的,或许就是权钱交易.
从恒大的破产到房地产经济下的泡沫可以看出政府的监管到处都是漏洞.
说没有官员从中获取什么我都不愿意相信.
